src/HOI4World/Map/RailwayTests.cpp

```diff
@@ -12,7 +12,7 @@ TEST(HoI4World_Map_Railway, LevelIsAsSet)
 {
 	const Railway railway(2, {1, 2});
 
-	EXPECT_EQ(railway.getLevel(), 2);
```

Comment:
Shouldn't expected value be first?

Reply:
Despite my original use, this is the usual order.